client/lib/analytics/with-tracking-tool/index.jsx

```diff
@@ -1,30 +1,24 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
Nit: maybe we can sort those imports alphabetically?

Reply:
They seem already sorted to me 🙂 Or should `@wordpress/*` be sorted before `react`?

Reply:
`createHigherOrderComponent` would be above `useDispatch` because `c` comes before `u`. It's a nit anyway, no need to open a new PR for that 😉

Reply:
I see, you sort by import name, not by library name 🙂 How would you sort this?
```js
import { aaa, zzz } from 'one';
import { bbb, yyy } from 'two';
```
Always by the first import?

Reply:
Yep, not by library name, just sorting the lines alphabetically. That's why the named `{` imports end up after the default `A-Z` ones, too.